schemas/event.yml

```diff
@@ -38,53 +38,174 @@
+            Events with `event.kind:metric` indicate that a measurement was taken.
+        - name: state
+          description: >
+            This value is similar to metric, except that the entity being measured does not
+            provide a numeric metric value, but rather one of a fixed set of conditions or states.
+            For example a periodic event reporting a "fin_wait" state of a TCP connection
+            on a host might use `event.type:state`.
+        - name: pipeline_error
+          description: >
+            This value indicates that an error occurred during the ingestion of this event,
+            and that event data may be missing, inconsistent, or incorrect.
+            `event.kind:pipeline_error` is often associated with parsing errors.
+        - name: signal
+          description: >
+            The signal value is used by Elastic Kibana apps, such as SIEM, for app-specific purposes.
+            `event.kind:signal` is thus reserved and should not be used for the ingestion
+            of events into Elasticsearch.
 
     - name: category
       level: core
       type: keyword
-      short: Event category.
+      short: Event category. (Categorization Field)
       description: >
-        Event category.
-
-        This contains high-level information about the contents of the event. It
-        is more generic than `event.action`, in the sense that typically a
-        category contains multiple actions.
-      example: user-management
-      accepted_values:
-      - name: authentication
-        description: >
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-          incididunt ut labore et dolore magna aliqua.
-        expected_event_types:
-          - allow
-          - deny
-          - info
-      - name: process
-        description: >
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-          incididunt ut labore et dolore magna aliqua.
-        expected_event_types:
-          - start
-          - info
-          - end
+        This is one of four ECS Categorization Fields, and indicates the
+        second level in the ECS category hierarchy.
+
+        `event.category` represents the "big buckets" of ECS categories. For example,
+        filtering on `event.category:process` yields all events relating to process activity.
+        This field is closely related to `event.type`, which is used as a subcategory.
+      example: authentication
+      allowed_values:
+        - name: authentication
+          description: >
+            Events in this category are related to the challenge and response process
+            in which credentials are supplied and verified to allow the creation of a session.
+            Common sources for these logs are Windows event logs, ssh logs, etc.
+            Visualize and analyze events in this category to look for unusual login activity,
+            failed logins, etc.
```

Comment:
Can we change this to say

> Visualize and analyze events in this category to look for authentication-related activity such as unusual or failed login attempts.

? Ending two sentences in a row in `etc.` makes me 😢